CONTRIBUTING.md

```diff
@@ -35,9 +38,32 @@ To contribute to this repository:
 
           $ git push -u origin my-new-branch
 
-Finally, go to the web page of your fork of the `everware` repo,
+Finally, go to the web page of your fork of the `everware` repository,
 and click 'Pull request' to send your changes to the maintainers for
-review. This will send an email to the committers.
+review. This will send an email to the committers. A valid PR includes
+the following items:
+
+* a description of the problem you are solving
+* a discussion why you made the choices you made
+* contains the minimal set of changes needed to solve the problem (do
+  not mix several issues, rather create separate PRs)
+* unit and frontend tests for your changes
```

Comment:
...or justification why tests are not applicable for this PR? (e.g. documentation changes like this one)